kitsune/sumo/jinja2/includes/google-analytics.html

```diff
@@ -0,0 +1,11 @@
+<!--
+Read more about our custom configuration and use of Google Analytics here:
+TODO: make sure we have a similar public comment about use on SUMO?
```

Comment:
We should probably have some kind of statement here for the curious/concerned.

Reply:
I recall seeing something like this on bedrock.. can you point me to where that is and I can add.

Reply:
The bug comment below is what we have on bedrock, and, re-reading it, it may apply here as well. (SUMO is mentioned in the bug comment.)

Maybe we can just drop the `TODO` above?